ddl/ttl.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022 PingCAP, Inc.
```

Comment:
1、remember to update bazel config when you create a new file.
2、ddl has been put too many files. so can you put them into ```ddl/ttl``` if it is easy to do ?

Reply:
1. Done
2. They depend on some internal structs / functions of ddl, for example `findColumnByName`, `ddlCtx`... So they cannot be moved to `ddl/ttl` directly :thinking: . 
   If it's an issue, I suggest to try to extract some utilities functions first, so that other functions would be easier to be extracted.